content/learn/book/welcome/plugins/_index.md

```diff
@@ -1,16 +1,96 @@
+       .init_resource::<Score>()
+       // Increments the score by 1 every pass of the game loop
+       .add_system(increment_score.system())
+       // Prints the current value of the score
+       .add_system(report_score.system());
+  }
+}
+
+#[derive(Default, Debug)]
+struct Score(u8);
+
+fn increment_score(score: ResMut<Score>){
+  score.0 += 1;
+}
+
+fn report_score(score: Res<Score>){
+  dbg!(score);
+}
+```
 
 ## `DefaultPlugins`
 
-## Writing your own plugins
+Bevy's `DefaultPlugins` are intended to get up and running with a "standard game" as quickly as possible. Let's take a look at the [source](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_internal/src/default_plugins.rs) again.
+
+As you can see, there are some added niceties (logging and diagnostics, scenes, input and windowing) as well as a number of plugins controlled by feature flags.
+These are heavier dependencies that you may want to disable for some games (or platforms), and replace with your own alternatives.
+To do so, we can use [cargo features](https://doc.rust-lang.org/cargo/reference/features.html) to disable them in our game.
+
+In your `Cargo.toml` file, disable default features and opt-in to the [features you want](https://github.com/bevyengine/bevy/blob/main/docs/cargo_features.md):
+
+```toml
+[dependencies]
+bevy = {"0.5", , default-features = false, features = ["..."]}
+```
+
+As shown in the [plugin_group.rs](https://github.com/bevyengine/bevy/blob/latest/examples/app/plugin_group.rs) example, you can also configure plugin groups from within Bevy itself.
+
+## `MinimalPlugins`
+
+If you're looking to structure your Bevy app in an unusual way and don't want to use most of the functionality provided by the engine, you can choose to use  Bevy's [`MinimalPlugins`](https://docs.rs/bevy/latest/bevy/struct.MinimalPlugins.html) instead.
+
+We can click through to the [source]((https://github.com/bevyengine/bevy/blob/latest/crates/bevy_internal/src/default_plugins.rs)) for the `impl PluginGroup for MinimalPlugins` to see that this adds [`CorePlugin`](https://docs.rs/bevy/latest/bevy/core/struct.CorePlugin.html) and [`ScheduleRunnerPlugin`](https://docs.rs/bevy/latest/bevy/app/struct.ScheduleRunnerPlugin.html).
+
+The `CorePlugin` handles low-level fundamentals such as updating app time, while the `ScheduleRunnerPlugin` sets up the main game loop to run repeatedly over time.
+This functionality is essential: starting with these plugins is virtually always going to be a safe bet.
 
 ## Third-party plugins
+
+Importing 3rd-party plugins is easy; they're just Rust code!
+
+1. Find a Bevy plugin (such as from our [collection of assets](https://bevyengine.org/assets/)).
+2. Add it to your `Cargo.toml` as a crate under `[dependencies]`.
+3. Import the code definitions from the crate.
+4. Add the plugin to your app!
+
+Follow the documentation and examples from the crates you're importing to make sure you have everything configured properly.
```

Comment:
I think it would be most relevant to include the content of #195's mention of the plug 'n playiness here (if not clear enough at line 15-17) along with what to do if the user wants to publish their own crate/plugin.

Reply:
Added in https://github.com/bevyengine/bevy-website/pull/176/commits/6615a69236db3ccc7a8d27c5a28754e61e41aa8f